sentry_sdk/client.py

```diff
@@ -103,6 +103,15 @@ class _Client(object):
     def __init__(self, *args, **kwargs):
         # type: (*Any, **Any) -> None
         self.options = get_options(*args, **kwargs)  # type: Dict[str, Any]
+
+        try:
+            project_root = os.getcwd()
+        except Exception:
+            project_root = None
+
+        self.configuration = {
```

Comment:
Probably makes sense. I put it in configuration because it can not be set but it is just read from os.getcwd(). But it is cleaner in options and also being able to set it would be cool. Will change.